ci/requirements-py36.yml

```diff
@@ -13,8 +12,7 @@ dependencies:
   - netcdf4
   - pytest
   - eofs
-  - cftime
-  - bottleneck
-  - matplotlib
-  - netcdf4
-  - lxml 
+  - lxml
+  - pip
+  - pip:
+    - xskillscore
```

Comment:
Curious if there's a difference if this is indented under pip: rather than just listed like the rest?

Reply:
The others are via conda. This is a pip-specific installation. I'm not sure if xskillscore can be installed through conda? Or maybe it can because it's on PyPI. I was just following esmlab.

Reply:
Ah got it. Didn't realize the rest were conda.